### ISSUE TITLE
204 No Content improvements

### ISSUE THREAD

**Issue by matthewmueller** [livebud/bud#10]
For example,

```go
func (c *Controller) Create() {
  // do something
}
```

These functions should return `204 No Content` if successful.

**Comment by theEyeD** [comment]
This seems to work for void controllers already. Changing the `Show` in the basic example results in a 204.

However, if the func returns a nil error, it is parsed as a 200. Not sure if it would make sense for void and nil error to parse the same.

**Comment by matthewmueller** [comment]
Thanks for looking into this!

> This seems to work for void controllers already.

Okay, so I think it's a bit more nuanced then. I saw an error when you only have one controller action and that controller action had no results. I think it's an "import is not used" error. Should be pretty easy to fix, just moving an `imports.AddNamed` to a different location in the controller loader.

> if the func returns a nil error, it is parsed as a 200

Yep, I totally think it makes sense for `func (c *Controller) Action(...) error { return nil }` to also return a `204 No Content`

**Comment by theEyeD** [comment]
> Okay, so I think it's a bit more nuanced then. I saw an error when you only have one controller action and that controller action had no results. I think it's an "import is not used" error. If so, should be pretty easy to fix, just moving an `imports.AddNamed` to a different location in the controller loader.

So I did a bit of playing around with different void scenarios, and the only scenario I found that didn't work was having a void action tied to a view. It resulted in:
```
# github.com/go-duo/bud/example/basic/bud/.app/controller
bud/.app/controller/controller.go:50:24: not enough arguments in call to i.View.Handler
        have (string)
        want (string, interface{})
```

> Yep, I totally think it makes sense for `func (c *Controller) Action(...) error { return nil }` to also return a `204 No Content`

Both of these are fixed in #68
